grails-app/migrations/0.8.x/changelog-2021-03-03-1330-create-table-invoice.xml

```diff
@@ -114,32 +114,9 @@
       </not>
     </preConditions>
     <createTable tableName="invoice_reference_number">
-      <column name="invoice_id" type="CHAR(38)"/>
+      <column name="invoice_reference_numbers_id" type="CHAR(38)"/>
       <column name="reference_number_id" type="CHAR(38)"/>
+      <column name="reference_numbers_idx" type="INT"/>
     </createTable>
   </changeSet>
-
-  <changeSet author="awalkowiak" id="0303202113300-8">
```

Comment:
We still need these foreign key references. 

invoice_reference_number.invoice_reference_numbers_id -> invoice.id
invoice_reference_number.reference_number_id -> reference_number.id